components/app-triplestore/src/main/java/org/trellisldp/app/triplestore/TrellisServiceBundler.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.app.triplestore;
+
+import static com.google.common.cache.CacheBuilder.newBuilder;
+import static java.util.concurrent.TimeUnit.HOURS;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import com.google.common.cache.Cache;
+
+import io.dropwizard.setup.Environment;
+
+import java.util.Optional;
+
+import org.apache.jena.rdfconnection.RDFConnection;
+import org.slf4j.Logger;
+import org.trellisldp.agent.SimpleAgentService;
+import org.trellisldp.api.AccessControlService;
+import org.trellisldp.api.AgentService;
+import org.trellisldp.api.AuditService;
+import org.trellisldp.api.BinaryService;
+import org.trellisldp.api.EventService;
+import org.trellisldp.api.IOService;
+import org.trellisldp.api.IdentifierService;
+import org.trellisldp.api.MementoService;
+import org.trellisldp.api.NamespaceService;
+import org.trellisldp.api.RDFaWriterService;
+import org.trellisldp.api.ResourceService;
+import org.trellisldp.api.ServiceBundler;
+import org.trellisldp.app.TrellisCache;
+import org.trellisldp.file.FileBinaryService;
+import org.trellisldp.file.FileMementoService;
+import org.trellisldp.id.UUIDGenerator;
+import org.trellisldp.io.JenaIOService;
+import org.trellisldp.namespaces.NamespacesJsonContext;
+import org.trellisldp.rdfa.HtmlSerializer;
+import org.trellisldp.triplestore.TriplestoreResourceService;
+import org.trellisldp.webac.WebACService;
+
+/**
+ * A triplestore-based service bundler for Trellis.
+ */
+public class TrellisServiceBundler implements ServiceBundler {
```

Comment:
Should configuration be available from this class?

Reply:
Can you clarify this? Do you mean whether the configuration should be accessible via some sort of `::getConfiguration` method?